scripts/install_olm.sh

```diff
@@ -75,6 +74,11 @@ function check_for_olm_components() {
 function check_existing_installation() {
   log separator
   echo "Checking for existing installation of Dell CSM Operator"
+  # get namespace from YAML file for deployment
+  NS_STRING=$(cat ${DEPLOYDIR}/operator_community.yaml | grep "namespace:" | head -1)
```

Comment:
use cat ${MANIFEST_FILE} instead of hardcoding

Reply:
tc